src/ingestors/zora-mint/index.ts

```diff
@@ -0,0 +1,102 @@
+import { MintContractOptions, MintIngestor, MintIngestorResources } from '../../lib/types/mint-ingestor';
+import { MintIngestionErrorName, MintIngestorError } from '../../lib/types/mint-ingestor-error';
+import { MintInstructionType, MintTemplate } from '../../lib/types/mint-template';
+import { MintTemplateBuilder } from '../../lib/builder/mint-template-builder';
+import { zoraMintAbi } from './abi';
+import {  getZoraMintPriceInEth,getZoraContractMetadata } from './onchain-zora';
+import { urlForValidZoraPage,zoraOnchainIdDataFromUrl } from './offchain-metadata';
+// import { getProhibitionContractMetadata} from './onchain-zora';
+// import { Address, encodeAbiParameters } from "viem";
+// 
+export class zoraIngestor implements MintIngestor {
+  // async supportsUrl(url: string): Promise<boolean> {
+  //   return new URL(url).hostname === 'zora.co';
+  // }
+  configuration = {
+    supportsContractIsExpensive: true,
+  };
+
+  async supportsUrl(_resources: MintIngestorResources, url: string): Promise<boolean> {
+    return new URL(url).hostname === 'zora.co' ;
```

Comment:
Will this work for 100% of mints across all chains on Zora?

If not, we should filter this to include only those that will.